internal/apps/msp/apm/browser/consumer.go

```diff
@@ -65,8 +71,11 @@ func (p *provider) invoke(key []byte, value []byte, topic *string, timestamp tim
 	metric.Tags["cid"] = parts[1]
 	metric.Tags["uid"] = parts[2]
 	metric.Tags["ip"] = parts[3]
-	if len(parts) > 5 {
-		metric.Tags["ai"] = parts[5]
+	if len(parts) > 5 && len(parts[5]) > 0 {
+		ai, err := base64.StdEncoding.DecodeString(parts[5])
```

Comment:
you'd better add a comprehensive comment for `invoke` method to explain each part, especially for part 6 `ai`.